NEWS.md

```diff
@@ -1,5 +1,10 @@
 # rsconnect (development version)
 
+* `deployApp()` and friends will now respect packrat and renv lock files,
```

Comment:
Do we report to the user during deploy how we're determining the bundle dependencies? (either by default, or only when using verbose logging?)

Reply:
I'll add. I think it makes sense to emit when `quiet = FALSE` (the default), since it's important for the average user.